.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: objective-c
+osx_image: xcode7
+
+env:
+  matrix:
+    - LC_CTYPE=en_US.UTF-8
+  global:
+    secure: kg5TFTNojZ9VOYHApONRUXSbwu+erLs3WSNNX/CKbkO8eQnBo+1koXHnEtOTQ641YRAVpFLX4QK7pbRfmWWWzR6ntNgXmvE4IUa1ggsUlcorApLq+cQin1Tp1es0/Ul0nYLJN8WXbcObKSLy8FOmHkrMWYuffMtH9e7qEL01ei0=
+
+
+before_install:
+  - ./script/import-certificates
+  - gem install cocoapods xcpretty -N
+  - brew update 
+  - brew install carthage
+  - carthage bootstrap --use-submodules --no-use-binaries
+
+script:
+  - pod spec lint APIKit.podspec
+  - set -o pipefail && xcodebuild test -workspace APIKit.xcworkspace -scheme APIKit-iOS -sdk iphonesimulator | xcpretty -c
```

Comment:
`set -o pipefail` should be effective in the current terminal session, so we could write as follows:

```yaml
- set -o pipefail
- xcodebuild test -workspace APIKit.xcworkspace -scheme APIKit-iOS -sdk iphonesimulator | xcpretty -c
- xcodebuild test -workspace APIKit.xcworkspace -scheme APIKit-Mac | xcpretty -c
```